dashboard/app/api.go

```diff
@@ -786,8 +789,8 @@ func reportCrash(c context.Context, build *Build, req *dashapi.Crash) (*Bug, err
 		if len(req.Report) != 0 {
 			bug.HasReport = true
 		}
-		for _, name := range newSubsystems {
-			bug.addSubsystem(BugSubsystem{name})
+		if len(newSubsystems) != 0 {
```

Comment:
When can the newSubsystems be empty? Is it only for namespaces that don't support subsystems? If yes, then I would call SetSubsystems unconditionally. There is no harm in replacing empty list of subsystems with empty list of subsystems.

Reply:
We (plan to) recalculate them on saved crashes, so it will also be empty if the reported crash won't be saved.